lib/tasks/one_off/clear_lower_tier_with_conviction_flags.rake

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+namespace :one_off do
+  desc "Clear LOWER tier registrations with conviction flags"
+  task clear_lower_tier_with_conviction_flags: :environment do
+    registration_ids = scope
+    registration_ids.each do |id|
+      registration = WasteCarriersEngine::Registration.find(id)
+      registration.conviction_sign_offs.destroy_all
+      registration.save
+    end
+  end
+
+  def scope
+    WasteCarriersEngine::Registration.collection.aggregate(lower_tier_conviction_pipeline).pluck(:_id)
+  end
+
+  def lower_tier_conviction_pipeline
```

Comment:
Can this be done using a single `where` clause with `tier: "LOWER"` and `convictionSignoffs.exists: true` ?

Reply:
This doesn't seem to work, I've tried 

```
    WasteCarriersEngine::Registration.where(
      tier: "LOWER",
      convictionSignOffs: { "$exists": true, "$ne": [] }
    ).pluck(:_id)
```
and

```
    WasteCarriersEngine::Registration.where(
      tier: "LOWER",
      :convictionSignOffs.ne => [],
      :convictionSignOffs.exists => true
    ).pluck(:_id)
```